test-utils/testlib/src/runtime_utils.rs

```diff
@@ -8,12 +8,16 @@ use near_chain_configs::Genesis;
 use near_primitives::account::Account;
 use near_primitives::hash::{hash, CryptoHash};
 use near_primitives::state_record::StateRecord;
-use near_primitives::types::{AccountId, MerkleHash, StateRoot};
+use near_primitives::types::{AccountId, Balance, MerkleHash, StateRoot};
 use near_store::test_utils::create_trie;
 use near_store::{Trie, TrieUpdate};
 use neard::config::GenesisExt;
 use node_runtime::{state_viewer::TrieViewer, Runtime};
 
+pub fn account_new(amount: Balance, code_hash: CryptoHash) -> Account {
```

Comment:
`init_test_account` reads better for me.

Reply:
init_* is already occupied by methods which are alter global static vars